web/src/ui/nodes/collapsible-card.ts

```diff
@@ -0,0 +1,53 @@
+import { LitElement, css, html } from 'lit'
+import { customElement, property } from 'lit/decorators'
+
+import { withTwind } from '../../twind'
+
+import './properties/generic/collapsible'
+
+/**
+ * UI Node Collapsible Details
+ *
+ * A component to render inside a node-card, which allows its content to be
+ * collapsed & hidden. This includes the necessary styling of the header &
+ * the shell of the body.
+ */
+@customElement('stencila-ui-node-collapsible-details')
```

Comment:
I'm don't fully understand why this is needed in addition to the existing `stencila-ui-node-collapsible-property`. But regardless, for consistency, this file should be move to `web/src/ui/nodes/properties/generic` folder and maybe renamed `collapsible-details.ts`.

Reply:
Done